test/unit/lib/plugins/aws/provider.test.js

```diff
@@ -474,30 +474,12 @@ role_arn = NOTDEFAULTWITHROLEROLE
       expect(awsCredentials.credentials.accessKeyId).to.equal('NOTDEFAULTKEYID');
     });
 
-    it('should get credentials from `provider.credentials`', async () => {
```

Comment:
Why do we remove that test?

Reply:
It should never be there in the first place as `provider.credentials` is not supported by schema. We can still introduce it to schema I guess, but we also have this issue that will fully deprecate and remove it: https://github.com/serverless/serverless/issues/8356

What do you think?

Reply:
I think we can have that test removed. It should also be deprecated in context of v2